tests/test_compat.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+
+try:
+    import OpenSSL
+except ImportError:
+    OPENSSL_AVAILABLE = False
```

Comment:
Definitely a nit, and I'd understand if you don't want to update it, but can this be `pyOpenSSL` since that's what is actually missing.

Reply:
Fixed